framework/src/transfers/MultiAppMeshFunctionTransfer.C

```diff
@@ -140,8 +129,12 @@ MultiAppMeshFunctionTransfer::transferVariable(unsigned int i)
           {
             if (bboxes[i_from].contains_point(*node + _to_positions[i_to]))
             {
+              // <system id, node id>
               std::pair<unsigned int, dof_id_type> key(i_to, node->id());
+              // <pid, <key>> --> point id
+              // point id is counted from zero
               point_index_map[i_proc][key] = outgoing_points[i_proc].size();
+              // pid --> points
```

Comment:
Same here: in the absence of the ability to write out a long arrow in a code comment, I would just say: `pid maps to points`